packages/calypso-build/webpack/minify.js

```diff
@@ -42,14 +42,23 @@ function chooseTerserEcmaVersion( browsers ) {
  */
 module.exports = ( options ) => {
 	let terserOptions = options.terserOptions || {};
+	const isDesktop = options.isDesktop || false;
+	const browsers = isDesktop ? [ 'last 2 Chrome versions' ] : supportedBrowsers;
 	terserOptions = {
-		ecma: chooseTerserEcmaVersion( supportedBrowsers ),
+		ecma: chooseTerserEcmaVersion( browsers ),
 		ie8: false,
-		safari10: supportedBrowsers.some(
-			( browser ) => browser.includes( 'safari 10' ) || browser.includes( 'ios_saf 10' )
-		),
+		mangle: true,
+		compress: ! isDesktop,
```

Comment:
How bigger are the final assets with this disabled?

Reply:
I'll take a peek - I need to figure out where in the application bundle to look.

These particular optimizations are recommended in the [Terser docs](https://github.com/terser/terser#terser-fast-minify-mode) for "fast minify" mode:

<p align="center">
<img width="600" alt="terser-fast-minify" src="https://user-images.githubusercontent.com/8979548/89913734-11faaa00-dbc2-11ea-9369-42cb838aec79.png">
</p>

Reply:
Some rather ... unexpected results when examining the application bundle. I rebased this branch on latest Calypso master and compared the bundle/application size of the app in master and in this branch (so the source code is the same, only the webpack settings are different.

Before optimizations: 234 MB
After optimizations: **215.9MB** (!)

This is surprising. It seems the aggregate of toggling these various optimizations is a bundle size that's _smaller_, despite the compression/minification settings being loosened.

Reply:
Surprising indeed. 

Where can I find the application bundles before and after? I'd like to understand why (maybe it is something we can use to make browser bundlers smaller too!)

Reply:
It's certainly an unexpected outcome. I'm not sure the difference is statistically significant, but at the very least the application bundle size before/after seem comparable. In a pure web environment, some of these tradeoffs may/may not make sense although it's certainly worth exploring.

To compare, I downloaded builds from master and from this branch, extracted and took a glance at the application size on disk. Clicking the green checkmark on the main Calypso landing page will reveal CI for the master branch:

<p align="center">
<img width="500" alt="wp-calypso-master-ci" src="https://user-images.githubusercontent.com/8979548/89950599-ee525680-dbf7-11ea-8a72-79f9933ec26c.png">
 </p>

To inspect an application bundle, download the .zip artifact for the job `wp-desktop-mac`. Extract and note the size of the bundle in Finder.

<p align="center">
<img width="500" alt="Screen Shot 2020-08-11 at 5 30 01 PM" src="https://user-images.githubusercontent.com/8979548/89950782-4d17d000-dbf8-11ea-9441-ecf9904f1722.png">
</p>